File: ex1.py
#!/usr/bin/env python3

import sys

print ("***Here is a python3.x example.")

if 1 >= len(sys.argv):
    print("(no args provided)")
else:
    for i in range(1,len(sys.argv)):
        print("argv[%d] = %s" % (i, sys.argv[i]))
    
s = input ("What is your name? ")
print ("WELL, " + s + " you have a lovely name")

File: ex1.java
import java.util.Scanner;

public class ex1 {
    public static void main (String argv[]) {
	System.out.printf ("Here is a java example%n");

	if (argv.length == 0) {
	    System.out.printf("(no args provided)%n");
	} else {
	    int i;
	    for (i = 0; i < argv.length; i++) {
		System.out.printf("argv[%d] is '%s'%n",i, argv[i]);
	    }
	}
	
 	Scanner sc = new Scanner (System.in);
	System.out.printf ("What is your name? ");
	String name = sc.nextLine ();
	System.out.printf ("Well, hello %s, what a lovely name\n", name);
    }
}
